architecture/nvidia-gpu-architecture-overview.adoc

```diff
@@ -51,17 +51,15 @@ include::modules/nvidia-gpu-red-hat-device-edge.adoc[leveloffset=+2]
 .Additional resources
 * link:https://cloud.redhat.com/blog/how-to-accelerate-workloads-with-nvidia-gpus-on-red-hat-device-edge[How to accelerate workloads with NVIDIA GPUs on Red Hat Device Edge]
 
-include::modules/nvidia-gpu-features.adoc[leveloffset=+1]
 
+include::modules/nvidia-gpu-features.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
-
 * link:https://docs.nvidia.com/ngc/ngc-deploy-on-premises/nvidia-certified-systems/index.html[NVIDIA-Certified Systems]
-* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/openshift/nvaie-with-ocp.html[NVIDIA AI Enterprise with OpenShift]
+* link:https://docs.nvidia.com/ai-enterprise/index.html#deployment-guides[NVIDIA AI Enterprise]
 * link:https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/overview.html#[NVIDIA Container Toolkit]
-* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/openshift/enable-gpu-monitoring-dashboard.html[Enabling the GPU Monitoring Dashboard]
-* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/mig-ocp.html[MIG Support in OpenShift Container Platform]
-* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/time-slicing-gpus-in-openshift.html[Time-slicing NVIDIA GPUs in OpenShift]
-* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/mirror-gpu-ocp-disconnected.html[Deploy GPU Operators in a disconnected or airgapped environment]
-* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/install-nfd.html[Installing the Node Feature Discovery (NFD) Operator]
-* link:https://docs.openshift.com/container-platform/4.13/hardware_enablement/psap-node-feature-discovery-operator.html#installing-the-node-feature-discovery-operator_node-feature-discovery-operator[{product-title} Installing the Node Feature Discovery Operator]
+* link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/enable-gpu-monitoring-dashboard.html[Enabling the GPU Monitoring Dashboard]
+* link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/mig-ocp.html[MIG Support in OpenShift Container Platform]
+* link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/time-slicing-gpus-in-openshift.html[Time-slicing NVIDIA GPUs in OpenShift]
+* link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/mirror-gpu-ocp-disconnected.html[Deploy GPU Operators in a disconnected or airgapped environment]
+* link:https://docs.openshift.com/container-platform/4.13/hardware_enablement/psap-node-feature-discovery-operator.html[Installing the Node Feature Discovery Operator]
```

Comment:
Is there a reason this internal one cannot be an xref?

Reply:
@jeana-redhat That was an original link that was not replaced in this bug. The link works. Should it be an xref?

Reply:
Yes, unless there is a specific reason that an `xref` would break things (iirc, service delivery has some exceptions because `xref`s would break them), we should be using `xrefs` for links between docs. 

See the [linking section in the docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references) for more info.

Reply:
@jeana-redhat Replaced the link url with xref. Please merge.

Reply:
@StephenJamesSmith @kcarmichael08 the `xref` is not constructed correctly. It uses `html` rather than `adoc` and is missing its anchor ID.

Currently:
```
* xref:../hardware_enablement/psap-node-feature-discovery-operator.html[Node Feature Discovery Operator]
```

Should be:
```
* xref:../hardware_enablement/psap-node-feature-discovery-operator.adoc#node-feature-discovery-operator[Node Feature Discovery Operator]
```

Not sure the best way to fix as it's been merged and cherrypicked. @kalexand-rh wdyt?

Reply:
@StephenJamesSmith, please open a PR to correct the xref.